includes/MagicWord.php

```diff
@@ -266,7 +271,17 @@ static function getCacheTTL( $id ) {
 	 */
 	static function getDoubleUnderscoreArray() {
 		if ( is_null( self::$mDoubleUnderscoreArray ) ) {
-			self::$mDoubleUnderscoreArray = new MagicWordArray( self::$mDoubleUnderscoreIDs );
+			self::$mDoubleUnderscoreArray = new MagicWordArray(
+				/**
+				 * Wikia change begin
+				 *
+				 * Allow different behavior for some magic words from a white list
+				 */
+				array_diff( self::$mDoubleUnderscoreIDs, self::$mWhitelistedIDs )
```

Comment:
This is the only place `$mDoubleUnderscoreIDs` is used, so why not just remove flags from `$mDoubleUnderscoreIDs` which you added earlier https://github.com/Wikia/app/blob/92b28ea61b0/includes/MagicWord.php#L180?